client/app/billing/payment/method/controller.js

```diff
@@ -0,0 +1,70 @@
+import { ALERTER_ID } from './constants';
+
+export default class BillingPaymentMethodCtrl {
+  /* @ngInject */
+  constructor($timeout, Alerter, OVH_PAYMENT_MEAN_STATUS, OVH_PAYMENT_METHOD_TYPE,
```

Comment:
Convention is to put the `constants` as the last deps injected

Reply:
:-)
I'm a little bit lost. In some other comment, we need to order injection in alphabetical way and here we need to put `constants` at the end :'(

Reply:
Angular deps first, then  services then constants. (but for each type you keep alphabetical order) 
Let's say I have `constructor($translate, MY_CONSTANTS, Alerter, $q, MyService`)
it will become `constructor($q, $translate, Alerter, MyService, MY_CONSTANTS)`
Hope it makes more sense now 😉